### ISSUE TITLE
rowcount does not work

### ISSUE THREAD

**Issue by pymssql-issue-migrator** [pymssql/pymssql#14]
_From [michele.simionato](https://code.google.com/u/114577181525321869099/) on July 01, 2010 02:22:45_

I have just tried on Linux with pymssql-1.9.908 the following script:

from pymssql import connect

CREATE = 'CREATE TABLE foo(i INTEGER PRIMARY KEY, t VARCHAR(100))'
DROP = 'DROP TABLE foo'
INSERT = 'INSERT INTO foo VALUES (&#37;d, &#37;s)'
UPDATE = 'UPDATE foo SET t=&#37;s WHERE i=&#37;d'
SELECT = 'SELECT \* FROM foo'
DELETE = 'DELETE FROM foo'

def main():
    conn = connect('itdbserver2', 'python', 'p1tone', 'data_manag')
    curs = conn.cursor()
    curs.execute(CREATE)
    try:
        curs.execute(INSERT, (1, 'line1'))
        print 'INSERT', curs.rowcount
        curs.execute(UPDATE, ('_line1_', 1))
        print 'UPDATE', curs.rowcount
        curs.execute(SELECT)
        print curs.fetchall()
        print 'SELECT', curs.rowcount
        curs.execute(DELETE)
        print 'DELETE', curs.rowcount
    finally:
        curs.execute(DROP)

I get:

INSERT 0
UPDATE 0
[(1, '_line1_')]
SELECT 1
DELETE 0

In other words, the rowcount after INSERT, UPDATE and DELETE is always 0
and this is wrong :-(

_Original issue: http://code.google.com/p/pymssql/issues/detail?id=10_
